app/test_main.py

```diff
@@ -1,3 +1,55 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestConvertToHuman:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected",
+        [
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (100, 100, [21, 17])
+        ]
+    )
+    def test_should_return_age_normal_range_values(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: int) -> None:
+        assert get_human_age(cat_age, dog_age) == expected
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected",
+        [
+            (0, 0, [0, 0]),
+            (-1, -1, [0, 0]),
+            (500, 600, [121, 117]),
+            (2345272727234, 34563473457, [586318181804, 6912694688])
+        ],
+    )
+    def test_should_return_age_edge_range_values(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: int) -> None:
+        assert get_human_age(cat_age, dog_age) == expected
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age",
```

Comment:
here you need to add what error you expect as the third argument.
this is how I did it, but maybe your option is correct, so you can ignore this comment)

Reply:
> here you need to add what error you expect as the third argument. this is how I did it, but maybe your option is correct, so you can ignore this comment)

We only have one type of an error in our case. In this case I'll have to put everywhere 3rd argument ```TypeError``` instead of one inside of the function as I did.